src/olympia/devhub/templates/devhub/suppressed_email.html

```diff
@@ -0,0 +1,10 @@
+{% if waffle.switch("suppressed-email") and request.user.suppressed_email %}
```

Comment:
`request.user.suppressed_email` doesn't exist for `AnonymousUser`. The failure is ignored because we're in a template, but it's good practice to avoid relying on that, avoids surprises in the long run. 
```suggestion
{% if waffle.switch("suppressed-email") and request.user.is_authenticated and request.user.suppressed_email %}
```

Also add a test that when the waffle is on the warning isn't shown to logged out users.

Reply:
Done